R/use.R

```diff
@@ -3,19 +3,24 @@
 #' This function will initialize {touchstone} in your package repository, use
 #' from root directory.
 #'
-#' @param overwrite Overwrites files if they exist.
+#' @inheritParams use_touchstone_workflows
 #' @details
 #' For more information see the 'Using touchstone' vignette:
 #'  `vignette("touchstone", package = "touchstone")
 #' @return
 #' The function is called for its side effects and returns `NULL` (invisibly).
-#' @example
+#' @examples
 #' \dontrun{
-#'    # within your repository
-#'    use_touchstone()
+#' # within your repository
+#' use_touchstone()
 #' }
+#' @seealso [touchstone::use_touchstone_workflows()]
 #' @export
-use_touchstone <- function(overwrite = FALSE) {
+use_touchstone <- function(overwrite = FALSE,
+                           on_comment = FALSE,
+                           command = "/benchmark",
```

Comment:
Do we need `on_comment` and `command` or should the user simply set `command` to `NULL` to signal *don't require a command*?

Reply:
I think the api is clearer with separate params but if you prefer too merge them :shrug:

Reply:
yes please. I agree it's taste though 😄